web/src/components/vis/LoadingsPlotTile.vue

```diff
@@ -51,13 +51,14 @@ export default {
 
 <template lang="pug">
 vis-tile(v-if="plot", title="PCA Loadings Plot", :loading="plot.loading")
-  loadings-plot(
-      :width="width",
-      :height="height",
-      :points="plot.data",
-      :pc-x="pcX",
-      :pc-y="pcY",
-      :show-crosshairs="showCrosshairs")
+  template(#default="wrapper")
```

Comment:
Is this change relevant to this PR?   It's fine if it isn't, just curious.

Reply:
I'm accessing the `wrapper.scale` one line below and I don't know another way to assign a scoped prop